cmd/srcd/cmd/defered_test.go

```diff
@@ -1,3 +1,5 @@
+// +build !integration,!regression
```

Comment:
no need for this build tags anymore

Reply:
Right! thank you!

Reply:
done [here](https://github.com/src-d/engine/pull/363/commits/62e72fe39ea2caec81986b09976332debcdb0d41).